src/H5Iint.c

```diff
@@ -1230,27 +1230,6 @@ H5I_dec_app_ref_always_close_async(hid_t id, void **token)
     FUNC_LEAVE_NOAPI(ret_value)
 } /* end H5I_dec_app_ref_always_close_async() */
 
-/*-------------------------------------------------------------------------
```

Comment:
The recent changes to not throw errors are no longer necessary, since the H5E package no longer increments IDs internally.